subsys/net/lib/lwm2m/ipso_temp_sensor.c

```diff
@@ -99,7 +99,7 @@ static void update_max_measured(uint16_t obj_inst_id, int index)
 			TEMP_MAX_MEASURED_VALUE_ID);
 }
 
-static int reset_min_max_measured_values_cb(uint16_t obj_inst_id)
+static int reset_min_max_measured_values_cb(uint16_t obj_inst_id, uint8_t *payload, uint16_t payload_len)
```

Comment:
Same here. Not to repeat myself, please check compliance check result (https://github.com/zephyrproject-rtos/zephyr/pull/30551/checks?check_run_id=1523239427) which lists all of the issues.